### ISSUE TITLE
ATLAS: Sentence-level translation alignment data

### ISSUE THREAD

**Issue by jacobwegner** [jhu-digital-manuscripts/AnIOp#46]

**Comment by jacobwegner** [comment]
@markpatton: @htpvu let us know that we should target getting some translation data in front of you sooner than later.

Regarding the sentence level alignment, I think whatever Web Annotation / Annotations we come up with to help address the translation-focused user stories would be similar to what we worked on together last fall:

https://github.com/scaife-viewer/hmt-cite-atlas/pull/2#issuecomment-561891031

A WA collection keyed off the folio identifier with each item in the collection having:
- Canvas / fragment information
- Text or HTML of the Greek content
- Text or HTML of the English content

We have that for both of the "targeted" sections within the _Iliad_.  We'll likely have a different endpoint, but I've generated links to relevant WAs from what we built in December:

- Lines 1.1-1.20 (starting with 12r):
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:12r/translation-alignment/0/html/
- Lines 2.474 - 2.760 (33v - 39r)
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:33v/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:34r/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:34v/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:35r/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:35v/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:36r/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:36v/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:37r/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:37v/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:38r/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:38v/translation-alignment/collection/html/0/
  - https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:39r/translation-alignment/collection/html/0/

**Comment by markpatton** [comment]
@jacobwegner  

Sorry for not getting back to you sooner. This looks good to me. I wonder if there is a need to distinguish the translation from the transcription, but I will leave that question to people working on the user interface side.

**Comment by jabrah** [comment]
I would prefer having `next`/`prev` property values use the full absolute URL, instead of just the path, as it seems to currently use. This way the other AnnotationPage is immediately resolvable without messing with URLs. 

For example, `"next": "https://explorehomer-atlas-dev3.herokuapp.com/wa/urn:cite2:hmt:msA.v1:34v/translation-alignment/collection/html/1/"` instead of `"next": "/wa/urn:cite2:hmt:msA.v1:34v/translation-alignment/collection/html/1/"`

**Comment by GroovinChip** [comment]
I agree with John, otherwise looks good to me

**Comment by jacobwegner** [comment]
I've broken out some of the information from this PR into a document that I think we can use to document any of the WAs built up for this project:

[AniOp ATLAS Documentation](https://docs.google.com/document/d/1XB0vYGxY1M420lmpJhLe2M3IfzcOQwBYFiwKGaDQ9ug/edit#)

The _aniop-atlas-staging.eldarion.com_ domain should be stable from here on out, and we'll treat it like a production instance; we can provide ephemeral instances to preview new features before they get rolled out to the staging server.

**Comment by jacobwegner** [comment]
@GroovinChip and @jabrah: The `next` and `prev` links were an oversight, as I think we'd discussed that before.

They will now include the full absolute URL as requested:

https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:49r/translation-alignment/collection/html/0/

```json-ld
{
    "@context": "http://www.w3.org/ns/anno.jsonld",
    "id": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:49r/translation-alignment/collection/html/1/",
    "type": "AnnotationPage",
    "partOf": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:49r/translation-alignment/collection/html/",
    "startIndex": 10,
    "items": [
        ...
    ],
    "prev": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:49r/translation-alignment/collection/html/0/"
}

**Comment by jacobwegner** [comment]
Resolved by [explorehomer-atlas/pull/30](https://github.com/scaife-viewer/explorehomer-atlas/pull/30)

Translation Alignments have been documented within [AniOp ATLAS Documentation](https://docs.google.com/document/d/1XB0vYGxY1M420lmpJhLe2M3IfzcOQwBYFiwKGaDQ9ug/edit).
